### PR TITLE
PLAT-27 Update Build and Publish

### PR DESCRIPTION
**Pull request by mark-onenetwork**
**WHY**

- Migrate to Github Action from Travis CI.
- Migrate to Github Package from Bintray.

**How to test:**
- Check the workflow in the [Actions](https://github.com/sealink/quicktravel_client_java/actions).

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      run: ./gradlew versionDisplay
+
+    - name: Build with Gradle
+      run: ./gradlew build
+
+    - name: Gradle report
+      run: ./gradlew jacocoTestReport coveralls
+
+    - name: Publish to GitHub Packages
+      if: github.ref == 'refs/heads/master'
```
Comment (jmortlock) [inline .github/workflows/gradle.yml:32]: Can we change this to publish on tags?

File: build.gradle [hunk 2]
```diff
@@ -107,27 +108,16 @@ publishing {
...
-        licenses = ['MIT']
-        vcsUrl = "https://github.com/sealink/quicktravel_client_java.git"
-        version {
-            name = project.version
-            desc = project.version
-            released  = new Date()
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = "https://maven.pkg.github.com/mark-onenetwork/quicktravel_client_java"
```
Comment (jmortlock) [inline build.gradle:114]: Can we change mark-onenetwork to sealink

**Comment by mark-onenetwork** [comment]
@jmortlock Comments addressed, please check.

**Comment by jmortlock** [comment]
Looking good, can you also update the maven badge in the README.

Can you also publish a version so I can see if I can pull it from another project, is there a UI or something to see whats published?

**Comment by mark-onenetwork** [comment]
>Can you also publish a version so I can see if I can pull it from another project, is there a UI or something to see whats published?

Do you mean the package page? You can see it on the right hand of the repo home page.
https://github.com/sealink/quicktravel_client_java/packages/945137

**Comment by mark-onenetwork** [comment]
>can you also update the maven badge in the README.

What was the badge like? Displays the download count?

**Comment by jmortlock** [comment]
Looks like there where 2, one for travis build status and another which links you to the latest published artifact

**Comment by mark-onenetwork** [comment]
Fixed, and it looks like:
Screen Shot 2021-08-18 at 11 16 47 AM

**Review by jmortlock — APPROVED** [review]

**Review by HellRok — APPROVED** [review]

**Pull request merged**